prow/plugins/verify-owners/verify-owners.go

```diff
@@ -177,15 +207,15 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 		err := ghc.CreateReview(org, repo, pre.Number, draftReview)
 		if err != nil {
-			return fmt.Errorf("error creating a review for invalid %s file%s: %v", ownersFileName, s, err)
+			return fmt.Errorf("error creating a review for invalid %s file%s: %v", ownersFileName, s, err.Error())
```

Comment:
nit: `%v` will stringify the errors anyway, right? So we don't need these changes

Reply:
Hmm, I thought so too, but I want to avoid the kind of error like in #8430 :
`json: cannot unmarshal string into Go struct field ClientError.errors of type struct { Resource string "json:\"resource\""; Field string "json:\"field\""; Code string "json:\"code\""; Message string "json:\"message,omitempty\"" }`
Maybe the fix should be in `github/client.go`?